WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -6115,7 +6118,7 @@ e_byzantium = {
 				color2={ 255 255 255 }
 				
 				aramaic = "Kharka"
-				khalka = "Kharka"
+				#khalka = "Kharka" #what culture should it be, khalan?
```

Comment:
Well, [Khalka Mongols](https://en.wikipedia.org/wiki/Khalkha_Mongols) do exist, but I don't know what this refers to in particular. Presumably it should be `khalan`, yes.

Reply:
Done.